quickcomm/views.py

```diff
@@ -69,5 +71,25 @@ def logout(request):
 
 
 def register(request):
-    # TODO implement register
-    return HttpResponse('Register Page')
+    if request.method == 'POST':
+        form = UserCreationForm(request.POST)
+        if form.is_valid():
+            user = form.save()
+            author = Author(user=user)
```

Comment:
The author has a bunch of fields that may be required, so the input from the form should include them. I believe the way that the code is written right now in #52, most fields are required.

Reply:
even though the fields are required, it still will create an author and just fill them in as blank. If we want to add those other fields to the form that can be done in another PR so that this one can get merged.